modules/ztp-image-based-upgrade-procedure-rollback.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
```

Comment:
Maybe the issue lies with the previous section? Remove the Additional resources heading?

![Screenshot from 2025-01-09 17-11-00](https://github.com/user-attachments/assets/739947b5-a501-4ce0-81a8-9d6ed26ef317)